packages/ckeditor5-emoji/src/emojirepository.ts

```diff
@@ -29,6 +30,16 @@ const SKIN_TONE_MAP: Record<number, SkinToneId> = {
 
 const BASELINE_EMOJI_WIDTH = 24;
 
+/**
+ * Object storing emoji support level by the operating system.
+ * Some emojis that are perceived as supported, are compound-emoji from previous version. For those emojis
+ * there has to be performed an additional width check to verify system support.
+ */
+const EMOJI_SUPPORT_LEVEL = {
+	'🫩': 16, // face with bags under eyes
+	'🫨': 15 // shaking head
```

Comment:
According to my comment about `adult-*` emoji. Perhaps we should store the latest supported `major.minor` combination. 

An alternative could be using only the `major` version when comparing versions when figuring out the latest supported version.

Reply:
Checking the `major.minor` versions as You suggested.